cli/command_snapshot_copy.go

```diff
@@ -0,0 +1,190 @@
+package cli
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/kopia/kopia/repo"
+	"github.com/kopia/kopia/snapshot"
+)
+
+var (
+	snapshotCopyCommand     = snapshotCommands.Command("copy", "Copy snapshot history from another user or host").Alias("cp")
+	snapshotCopyDryRun      = snapshotCopyCommand.Flag("dry-run", "Do not actually copy snapshots, only print what would happen").Short('n').Bool()
+	snapshotCopyMove        = snapshotCopyCommand.Flag("move", "Delete original copied snapshots").Bool()
```

Comment:
UI: Should `move` be a flag or a separate command that shares the flags and the implementation?

Reply:
done